TransactionalSession public documentation (#5813)
Particular/docs.particular.net: nservicebus/transactional-session/ts-outbox-warning.include.md

> WARN: In order to guarantee atomic consistency across message and database operations, the [Outbox](/nservicebus/outbox) needs to be enabled. Otherwise `Commit` will only execute the database operations and the message sending and publishing will happen afterwards in a best-effort manner.

Reviewer: ```suggestion
WARN: In order to guarantee atomic consistency across message and database operations, the [Outbox](/nservicebus/outbox) needs to be enabled. Otherwise `Commit` dispatches message operations after executing all database operations.
```

> WARN: In order to guarantee atomic consistency across message and database operations, the [Outbox](/nservicebus/outbox) needs to be enabled. Otherwise `Commit` will only execute the database operations and the message sending and publishing will happen afterwards in a best-effort manner.

Reviewer 2: ```suggestion
WARN: In order to guarantee atomic consistency across message and database operations, the [Outbox](/nservicebus/outbox) needs to be enabled. Otherwise `Commit` executes database modifications first and then messages are dispatched with best-effort.
```
